pull_alerts.py

```diff
@@ -113,27 +122,40 @@ def get_formatted_incidents(recent_incidents):
     return formatted_incidents
 
 
+def _tag_incident(incident, tag_stats):
+    tagged = False
+    for idx, tag in enumerate(TAGS):
+        found_tag = any(tag.tag in note for note in incident.notes)
+        if not found_tag:
+            continue
+        tagged = True
+        tag_stats[idx] += 1
+    return tagged
+
+
 def print_stats(all_incidents, include_stats):
     if not include_stats:
         return
-    actionable = 0
-    non_actionable = 0
+
+    stats_table = PrettyTable()
+    stats_table.field_names = ["Incidents", "Number"]
+    stats_table.align["Incidents"] = "l"
+    stats_table.align["Number"] = "r"
+
+    # tag_stats is indexed the same way as TAGS so tag_stats[0] is #a, etc.
+    tag_stats = [0] * len(TAGS)
```

Comment:
can `tag_stats` be a dict keyed on tag? I think it'll be easier to understand that two lists you zip together

Reply:
Something like a dictionary with key = Tag and value is the number of pages? 

I was trying to optimize for not having to create all the tags in the function, but I just realized that this is a one time script, so it doesn't matter. I'll make the change.

Reply:
you can use `defaultdict(lambda: 0)` or `collections.Counter()` to make it read even simpler

Reply:
Oh counter is perfect! Thanks for the suggestion.